integrations/tensorflow/e2e/README.md

```diff
@@ -52,23 +53,17 @@ preferred.
 
 ```shell
 # Run math_test on all backends.
-bazel run :math_test_manual
+bazel run integrations/tensorflow/e2e:math_test_manual
```

Comment:
should this be //integrations for clarity that it is an absolute path?

Reply:
Sure! I just usually omit the `//` because blazel tab completion tends to time out on my cloud machine, for... whatever reason.